src/settings/MappingProfiles/detailsSections/constants.js

```diff
@@ -7,8 +7,8 @@ export const TRANSLATION_ID_PREFIX = 'ui-data-import.settings.mappingProfiles.ma
 export const FIELD_NAME_PREFIX = 'profile.mappingDetails.mappingFields';
 
 export const CREATE_INVENTORY_TYPES = {
-  INSTANCE_HOLDINGS_ITEM: 'Instance, holdings, item',
-  INSTANCE_HOLDINGS: 'Instance, holdings',
+  INSTANCE_HOLDINGS_ITEM: 'Instance, Holding, Item',
+  INSTANCE_HOLDINGS: 'Instance, Holding',
```

Comment:
I have questions:
1. The Jira the _UI_ should have the values "Instance, Holding, Item"; "Instance, Holding"; "Instance"; "None". The fact that this change is taking place here rather than in `translations/ui-data-import/en.json` suggests these values are not correctly i18n'ed.
2. Are we confident about "holding" rather than "holdings" or "holdings record"? The [FOLIO glossary](https://wiki.folio.org/display/FOLIOtips/Glossary) uses "holdings". Additionally, the [FOLIO standard for text case](https://ux.folio.org/docs/guidelines/language-rules/text-case-rules/) indicates that only "Instance" should be capitalized here.

Reply:
Hi @zburke this const is values sending to BE, UI labels are rendered as translations, e.g. https://github.com/folio-org/ui-data-import/blob/master/src/settings/MappingProfiles/detailsSections/edit/OrderDetailSection/EResourcesDetails.js#L66-L80